jobs/furnishings/tests/unit/stage_processors/test_post_processor.py

```diff
@@ -79,10 +79,11 @@ def test_processor(app, session, sftpserver, sftpconnection, test_name, furnishi
     processor = PostProcessor(app, furnishing_dict)
     with sftpserver.serve_content({app.config.get('BCLAWS_SFTP_STORAGE_DIRECTORY'): {}}):
         with patch.object(processor, '_bclaws_sftp_connection', new=sftpconnection):
-            processor.process()
-            # assert xml file is uploaded
-            with sftpconnection as sftpclient:
-                assert len(sftpclient.listdir(app.config.get('BCLAWS_SFTP_STORAGE_DIRECTORY'))) == 1
+            with patch.object(flags, 'is_on', return_value=False):
```

Comment:
Probably not the best way to do this. I'm basically trying to make sure the `post_processor` runs for this test even if the feature flag is preventing it. 

I could potentially refactor the `process` function so that it takes `flags` as a parameter (like `processor.process(flags=...)`) to make it easier to test.

Any thoughts?

Reply:
If it's not too much work, I think refactoring the function would be awesome.

Otherwise, I'm OK with keeping it as is.

Reply:
It's ok to use patch method. And another way to make the `post_processor` run in debugging/test is to add the flag value you want into `flags.json` file

Reply:
I'll give the flags.json a try! The only reason I'd like to avoid using patch here is that it'll return False for any flag value. So in the future if we add other flags this patch might not work.